.gitignore

```diff
@@ -22,6 +22,15 @@ data/processed/pubmed/*
 data/raw/open-access/oa-roadoi.rds
 data/ctgov-2018/ctgov/*
 data/processed/codebook-structure.csv
+data/raw/euctr_euctr_dump-2023-12-02-002347.csv
+TRN(registry data).rds
+data/raw/euctr_data_quality_results_scrape_feb_2024.csv
+euctr_euctr_dump-2024-02-03-054239.csv
```

Comment:
For some reason, the version of IV I cloned doesn't have a data/raw/registries folder, even though my gitignore does mention a data/raw/registries. I do have data/**processed**/registries though. Should I make a new folder in data/raw, and also a new data/raw/registries/euctr folder?

Reply:
@lucfernandez the `data/raw/registries` folder isn't showing up because the whole folder is ignored, so you have to create this folder and the `euctr` subfolder where you store the EUCTR files (and update the file paths accordingly). If you want to save any outputs to a `data/processed/euctr` folder you would also need to create that folder. See a tree of the folder structure in the readme: https://github.com/maia-sh/intovalue-data